src/pydocstyle/checker.py

```diff
@@ -54,6 +55,27 @@ class ConventionChecker:
     D40x: Docstring content issues
 
     """
+    _RE_ARGSTART = re(r"(^ {8})(\**[\w\[\]_]{1,}?)\s*?:(.{2,})?", IGNORECASE)
+
+    SECTIONS_WITH_ARGS = [
+        'Args',
+        'Arguments',
+        'Raises'
+    ]
+
+    BLOCK_ORDER = [
+        'Example',
+        'Examples',
+        'Args',
+        'Arguments',
+        'Return',
+        'Returns',
+        'Yield',
+        'Yields',
+        'Raises',
+        'Note',
+        'Notes'
+    ]
```

Comment:
Optional: Would we want to go a little bit more strict here to avoid confusion and increase alignment?

```suggestion
    BLOCK_ORDER = [
        'Example',
        'Args',
        'Returns',
        'Yields',
        'Raises',
        'Note',
    ]
```

Now this should be exactly as described in our Notion on docstrings.

Reply:
Can do!